examples/agent-sandbox-rl/agent_sandbox_rl/fleet.py

```
@@ -200,6 +214,14 @@ def overcommit_guard(self, expected: int | None = None):
     hard = self.config.max_live_sandboxes
     if expected is None:
       expected = self.plan_.total_replicas if self.plan_ else 0
+    if self.config.adopt_existing:
+      # The intent-based ceiling is meaningless here and would be actively
+      # misleading: an adopted plan's `total_replicas` is the provisioner's depth
+      # (large), while `live_owned_count` sees only pods carrying THIS run's
+      # label — of which adoption creates none. Ceiling × 1.5 against a count
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)